src/policies/list/components/PolicyEnactmentsDrawer/PolicyEnactmentsDrawer.tsx

```diff
@@ -91,7 +94,7 @@ const PolicyEnactmentsDrawer: React.FC<PolicyEnactmentsDrawerProps> = ({
               title={
                 <>
                   <TabTitleIcon>{tabData.icon}</TabTitleIcon>
-                  <TabTitleText> {tabData.title} </TabTitleText>
+                  <TabTitleText> {t(tabData.title)} </TabTitleText>
```

Comment:
I am not sure if this is gonna work. I think I was fixing the same issue in our other repo. Did you test it if it's gonna work? :)

Reply:
the translation is here: https://github.com/nmstate/nmstate-console-plugin/pull/16/files/97af920a7dce3c99bf25b55f7575cb7ab931627b#diff-08f8e7bbddc085fd6df6941f5c66b8e6ab9602064494b748bbaa1226f0b7f7b9R1

Title can only be one of those

Reply:
It works. Tested it.